File: Labs/lab04/max_2d_array.js
function maximum(array){
    let max_value = array[0][0];
    for(let i = 0; i < array.length; ++i){
        let in_array = array[i]
        for (let j = 0; j < in_array.length; ++j){
            if (in_array[j]>max_value){
                max_value=in_array[j]
            }
        }
    }
    return max_value
}

const arr = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
const m = maximum(arr)
console.log (m)

File: Labs/lab04/max_2d_array.py
def maximum(array):
    max_val = array[0][0]
    for i in range(len(array)):
        in_array = array[i]
        for j in range(len(in_array)):
            if in_array[j] > max_val:
                max_val = in_array[j]
    return max_val

arr = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
m = maximum(arr)
print(m)  
